config/routes.rb

```diff
@@ -0,0 +1,3 @@
+UiComponents::Engine.routes.draw do
```

Comment:
So far it is only done in the dummy app and I think that is enough, since we can now just run `rake styleguide` to run it via the dummy app.